index.html

```diff
@@ -1,29 +1,265 @@
+                        <li class="last-lang"><a href="#">Bootstrap</a></li>
+                        <li class="last-lang"><a href="#">Ruby</a></li>
+                    </ul>
+                   <div class="project-button">
+                    <a href="#">
+                    <img src="icons/see2.png"></a>
+                   </div>
+                </li>
+            </ul>
+            <div class="background">
+                <img src="icons/background.png">
+            </div>
+            <ul class="container">
+                <li class="grid-container">
+                    <div class="work-text1">
+                        Profesional Art Printing Data
+                    </div>
+                    <div class="work-text2">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                    </div>
+                    <ul class="new-lang">
+                        <li class="last-lang"><a href="#">HTML</a></li>
+                        <li class="last-lang"><a href="#">Bootstrap</a></li>
+                        <li class="last-lang"><a href="#">Ruby</a></li>
+                    </ul>
+                   <div class="project-button">
+                    <a href="#">
+                    <img src="icons/see2.png"></a>
+                   </div>
+                </li>
+            </ul>
+            <div class="background">
+                <img src="icons/background.png">
+            </div>
+            <ul class="container">
+                <li class="grid-container">
+                    <div class="work-text1">
+                        Profesional Art Printing Data
+                    </div>
+                    <div class="work-text2">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                    </div>
+                    <ul class="new-lang">
+                        <li class="last-lang"><a href="#">HTML</a></li>
+                        <li class="last-lang"><a href="#">Bootstrap</a></li>
+                        <li class="last-lang"><a href="#">Ruby</a></li>
+                    </ul>
+                   <div class="project-button">
+                    <a href="#">
+                    <img src="icons/see2.png"></a>
+                   </div>
+                </li>
+            </ul>
+     </section>
+     <section class="about">
+        <h3 class="about-me">About me</h3>
+        <div class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </div>
+            <div class="resume">
+            <ul>
+                <li><a href="#">Get my resume</a></li>
+            </ul>
```

Comment:
- [ ] Please follow the button interaction styles for the following button.

![image](https://user-images.githubusercontent.com/62290545/235460165-6c846f44-41a2-481c-aaac-6fdd8cc276ca.png)